.buildkite/scripts/generate_auditbeat_pipeline.sh

```diff
@@ -17,58 +17,67 @@ steps:
 
     steps:
       - label: ":ubuntu: Unit Tests"
-        command: "cd ${BEATS_PROJECT_NAME} && mage unitTest"
+        command: "cd $BEATS_PROJECT_NAME && mage build unitTest"
         notify:
           - github_commit_status:
               context: "Auditbeat: linux/Unit Tests"
         agents:
           provider: "gcp"
           image: "${IMAGE_UBUNTU_X86_64}"
           machineType: "${GCP_DEFAULT_MACHINE_TYPE}"
-        artifact_paths: "${BEATS_PROJECT_NAME}/build/*.*"
```

Comment:
I see some places where we do a single `*.*` and others with explicit paths in the pipeline files.

Reply:
fixed